app/views/root/_filter.html.erb

```diff
@@ -0,0 +1,81 @@
+<%
+  assignees = [{text: "All assignees", value: ""}]
+  content_types = []
+  states = []
+
+  @presenter.available_users.map do | user |
+    if user.id.to_s == @presenter.assigned_to
+      assignees << {text: user.name, value: user.id, selected: "true"}
+    else
+      assignees << {text: user.name, value: user.id}
+    end
+  end
+
+  format_filter_selection_options.map do | format |
```

Comment:
The `format_filter_selection_options` method is only called here, so we could move this functionality to that method to avoid doing two loops (obviously we would need to also add a `content_type` parameter, and pass that in here). We should probably also rename the method to `content_type_filter_selection_options`.

We could even update the `FilteredEditionsPresenter` to just return the list that we want in the first place.

If we move all these loops to the `FilteredEditionsPresenter`, then we also avoid the need to add some of those extra methods that have been added in this PR.